Packs/CortexVulnerabilityManagement/Playbooks/Cortex_VM_-_Vulnerability_Issue.yml

```diff
@@ -123,9 +123,9 @@ tasks:
 
         This automation runs using the default Limited User role, unless you explicitly change the permissions.
         For more information, see the section about permissions here:
-        For Cortex XSOAR 6.13, see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.13/Cortex-XSOAR-Administrator-Guide/Automations
-        For Cortex XSOAR 8 Cloud, see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8/Cortex-XSOAR-Cloud-Documentation/Scripts
-        For Cortex XSOAR on-prem, see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8.7/Cortex-XSOAR-On-prem-Documentation/Scripts
+        For Cortex.13, see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.13/Cortex-XSOAR-Administrator-Guide/Automations
```

Comment:
@ShiratG , FYI, I created a [bug](https://jira-dc.paloaltonetworks.com/browse/CIAC-12759) for this continuing to happen.  I can fix this manually if you please or we can wait the more permanent fix